edit-test-uid.js

```diff
@@ -0,0 +1,42 @@
+import * as firebase from 'firebase';
+import 'firebase/firestore';
+
+async function main() {
```

Comment:
Could you add a comment describing when this is useful, plox? I.e. something like `Use this script to update your member doc and related operations (imported from prod) to your auth profile in the test environment.`